index.js

```diff
@@ -0,0 +1,84 @@
+const express = require("express");
+const morgan = require("morgan");
+const allContacts = require("./data/contacts");
+const allMettings = require("./data/meetings");
+const contacts = allContacts.contacts;
+const meetings = allMettings.meetings;
+const app = express();
+
+app.use(express.json());
+app.use(morgan());
+
+app.get("/contacts", (req, res) => {
+  res.json({ contacts: contacts });
+});
+
+app.get("/contacts/:id", (req, res) => {
+  const { id } = req.params;
+  const contact = contacts.find((c) => c.id === Number(id));
+  if (!contact) {
+    return res.status(404).json({ message: "not found" });
+  }
+  res.json(contact);
+});
+
+app.delete("/contacts/:id", (req, res) => {
+  const { id } = req.params;
+  contacts.forEach((item, index) => {
+    if (item.id === id) {
+      return counters.splice(index, 1);
+    }
+  });
```

Comment:
The return wont stop an array method from continuing. Switch to a for, for of, or Array.find and Array.indexOf